src/components/map/CuriosityMap.tsx

```diff
@@ -1,9 +1,26 @@
-import React from "react";
+import React, {useState} from "react";
+import './Map.scss'
 
 export const CuriosityMap: React.FunctionComponent = () => {
+  const [date, setDate] = useState('0')
+
+  function handleClick(day:string) {
+    setDate(day)
+    console.log(date)
```

Comment:
Semicolons:
```suggestion
    setDate(day);
    console.log(date);
```
Also, I think you've mixed up some variable names! Is it `day` or `date`?